app/mailers/user_mailer.rb

```diff
@@ -204,4 +204,11 @@ def set_instance
   def locale
     @resource.locale.presence || I18n.default_locale
   end
+
+  def mail_with_new_layout(*, **)
+    mail(*, **) do |format|
```

Comment:
This is invalid syntax, you could do something like:
```suggestion
  def mail_with_new_layout(...)
    mail(...) do |format|
```

Reply:
I wrote this, and `(*, **)` is valid since Ruby 3.2 I think, but not before.

Same for `...`, but this one does not work even on 3.2 because it also passes the block, and Ruby fails because you then override the block

Reply:
Oops, sorry about that. Then the following should work:
```suggestion
  def mail_with_new_layout(*args, **kwargs)
    mail(*args, **kwargs) do |format|
```